mewarpx/tests/test_particle_scraping_stats.py

```diff
@@ -0,0 +1,95 @@
+"""
```

Comment:
This file can now be removed, right?

Reply:
It hasn't been copied anywhere else, so do we not need a test for this?

Reply:
The same functionality is tested in _WarpX/Examples/Modules/ParticleBoundaryScrape/PICMI_inputs_scrape.py_ so we are good.